### ISSUE TITLE
Pictures not showing on this site

### ISSUE THREAD

**Issue by zanqi** [getlantern/lantern#4194]
[Link](http://lifehacker.com/the-best-things-to-buy-at-the-dollar-store-1774228189)

Pictures on the articles (like above link) from Lifehacker can not be displayed when using Lantern. "Proxy all traffic" can fix it but it shouldn't come to that.

**Comment by myleshorton** [comment]
Thanks for the report @zanqi we'll give it a look! Sounds like a possible issue with detour @fffw?

**Comment by fffw** [comment]
Yeah during my test it looks like that detour successfully loaded a few images at i.kinja-img.com, added it to direct accessible sites, but further direct access from browser triggered `net::ERR_CONNECTION_RESET`. To overcome it, we have to let all traffic to go through Lantern.

Maybe it's time to check carefully if the performance gain of adding direct accessible sites to PAC pay off the potential issues like this, and the effort to flap PAC.

Another similar issue is that after disconnected VPN, those blocked sites accessible through VPN will never go through Lantern again.
